clang/test/CXX/drs/dr28xx.cpp

```diff
@@ -0,0 +1,41 @@
+// RUN: %clang_cc1 -std=c++20 -verify=expected %s
+
+namespace dr2847 { // dr2847: 19
+
```

Comment:
As this test requires C++20, it should be wrapped in `#if  __cplusplus`.

Reply:
`#if __cplusplus >= 202302L` skips over C++20, which doesn't seem intentional given the previous iteration of this PR I reviewed. I also put this comment on this specific lines, as we put `#if __cplusplus` checks inside namespace each test have, so that they are as independent as possible.